source/releasenotes/about.rst

```diff
@@ -17,128 +17,37 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is a 4.19 LTS minor release with over 300 fixes
-and improvements since the 4.19.0.0 release. Some of the highlights include:
-
-• Improvements for VMware migration to KVM
-• Support to manage/unmanage DATA volume of a primary storage on KVM
-• Support for NFS mount options for a primary storage on KVM
-• Support to change storage pool scope from Cluster to Zone and vice versa
-• Support for RHEL/OL/Rocky/Alma Linux in the same cluster
-• Import from remote KVM enhancements
-• Storage plugins, PowerFlex, StorPool, Linstor related fixes and improvements
-• Some CKS, Veeam (B&R) related fixes and improvements
-• Several UI fixes and improvements
-
-The full list of fixes and improvements can be found in the project release notes at
-https://docs.cloudstack.apache.org/en/4.19.1.0/releasenotes/changes.html
-
-What's in since 4.19.0.0
-========================
-
-Apache CloudStack 4.19.0.0 is the initial 4.19 LTS release with 300+ new
-features, improvements and bug fixes since 4.18, including 26 major
+Apache CloudStack 4.20.0.0 is the initial 4.20 LTS release with 190+ new
+features, improvements and bug fixes since 4.19, including 15 major
 new features. Some of the highlights include:
 
-• CloudStack Object Storage Feature
-• VMware to KVM Migration
-• KVM Import
-• CloudStack DRS
-• OAuth2 Authentication
-• VNF Appliances Support
-• CloudStack DRS
-• CloudStack Snapshot Copy
-• Scheduled Instance Lifecycle Operations
-• Guest OS Management
-• Pure Flash Array and HPE-Primera Support
-• User-specified source NAT
-• Storage Browser
-• Safe CloudStack Shutdown
-• New CloudStack Dashboard
-• Domain migration
-• Flexible tags for hosts and storage pools
-• Support for Userdata in Autoscale Groups
-• KVM Host HA for StorPool storage
-• Dynamic secondary storage selection
-• Domain VPCs
-• Global ACL for VPCs
-
-The full list of new features can be found in the project release notes at
-https://docs.cloudstack.apache.org/en/4.19.0.0/releasenotes/changes.html
-
-.. _guestosids
-
-Possible Issue with volume snapshot revert with KVM
-===================================================
-
-Between versions 4.17.x, 4.18.0 and 4.18.1, KVM volume snapshot backups were
-not full snapshots and they rely on the primary storage as a backing store.
-To prevent any loss of data, care must be taken during revert operation and
-it must be ensured that the source primary storage snapshot file is present
-if the snapshot is created with any of these CloudStack versions.
-
-Users will have a backing store in their volume snapshots in the following cases:
-
-- the snapshots are from a ROOT volume created from template;
+• Webhooks
+• Dynamic and Static Routing
+• Ceph RGW Object Store Support
+• NSX integration
+• Shared Filesystem
```

Comment:
Shared FileSystems is the official name we are using for the feature. That's how it shows up in the UI.
Thanks.

```suggestion
• Shared Filesystems
```